aiida/tools/importexport/common/archive.py

```diff
@@ -79,9 +81,9 @@ def unpack(self):
         if os.path.isdir(self.filepath):
             extract_tree(self.filepath, self.folder)
         elif tarfile.is_tarfile(self.filepath):
-            extract_tar(self.filepath, self.folder, silent=True, nodes_export_subfolder=NODES_EXPORT_SUBFOLDER)
+            extract_tar(self.filepath, self.folder, silent=False, nodes_export_subfolder=NODES_EXPORT_SUBFOLDER)
```

Comment:
why change the default here?

Reply:
It's not the default. It's the actual call to `extract_tar` and `extract_zip`.
I think what would be better is to add `silent` as an ´Archive` property - but then again, it doesn't _really_ make sense to have an `Archive` be `silent` - only the action of unpacking it makes sense to be silent ... But maybe that's too philosphical, and we need a more practical approach? 😅